src/odemis/acq/fastem.py

```diff
@@ -398,9 +398,11 @@ def get_abs_stage_movement(self):
 
         # Move in the negative x direction, because the second field should be right of the first.
         # Move in positive y direction, because the second field should be bottom of the first.
```

Comment:
These 2 lines should be updated, to match the line below (and maybe remind which stage you expect to move... if it's not stage-scan).

Reply:
Ok. I'd even clarify like:
```
With role="stage" move pos in x and neg in y.
With role="stage-scan" move neg in x and pos in y (because this stage follows a referential where X and Y move in the global coordinates + a small extra angle)
```

Reply:
I updated the comment to clarify. I left out the part in the brackets. I think the "stage-bare" moves in the global referential and the "stage-scan" in the multiprobe referential. That's how we defined it in the yaml file.